tck/README.adoc

```diff
@@ -404,77 +404,44 @@ following in your pom.xml:
                 <configuration>
                     <redirectTestOutputToFile>true</redirectTestOutputToFile>
                     <suiteXmlFiles>
-                        <suiteXmlFile>tck-base-suite.xml</suiteXmlFile>
+                        <suiteXmlFile>${project.build.directory}/tck-suite/suites/tck-build-suite.xml</suiteXmlFile>
+                        <suiteXmlFile>${project.build.directory}/tck-suite/suites/tck-base-suite.xml</suiteXmlFile>
                     </suiteXmlFiles>
                     <forkCount>1</forkCount>
                 </configuration>
             </plugin>
+            
+            <!-- Extract the TCK Suite Files -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>unpack</id>
+                        <phase>process-test-classes</phase>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>org.eclipse.microprofile.jwt</groupId>
+                                    <artifactId>microprofile-jwt-auth-tck</artifactId>
+                                    <version>1.2</version>
+                                    <type>test-jar</type>
+                                    <overWrite>false</overWrite>
+                                    <outputDirectory>${project.build.directory}/tck-suite</outputDirectory>
+                                    <includes>**/tck-*-suite.xml</includes>
+                                </artifactItem>
+                            </artifactItems>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
         </plugins>
     </build>
 ```
 
-and then either copy the tck-base-suite.xml file from the TCK source tree into your
```

Comment:
I guess we need to keep this text for the moment with the minor update as the implementations may still keep cloning it so that they can exclude something, etc.  After your fragment showing how to have it extracted automatically, update `and then either copy the tck-base-suite.xml...` to something like `Alternatively, either copy the tck-base-suite.xml...`